content/en/releases/release.md

```diff
@@ -4,7 +4,7 @@ type: docs
 auto_generated: true
 ---
 
-<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in k/website -->
+<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in kubernetes/website -->
```

Comment:
I guess this content is generated by a script?

Reply:
Yep, https://github.com/kubernetes/website/pull/40499